src/Symfony/Component/Routing/Tests/Fixtures/glob/php_dsl_bar.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Symfony\Component\Routing\Loader\Configurator;
+
+return function (RoutingConfigurator $routes) {
+    $collection = $routes->collection();
+
+    $collection->add('bar_route', '/bar')
+        ->defaults(['_controller' => 'AppBundle:Bar:view']);
```

Comment:
`array()` instead of `[]` (same below, see fabbot patch)

Reply:
Sorry, old habits. Fixed.